### DISCUSSION TITLE
Default media section

### DISCUSSION THREAD

**Discussion by PtolemyIV** [punxaphil/custom-sonos-card#124] [General]
Firstly, absolute awesome panel card and is the first time I've seriously considered being able to use home assistant rather than sonos or spotify apps directly.

One question - is it possible to select a default media location? Most of our playlists are set up on Spotify primarily so ideally the default location would be the spotify playlists section rather than a sonos favourites one..

Obviously a killer feature would be to somehow incorporate a spotify search bar at the top but appreciate may be more challenging!

**Comment by punxaphil** [comment]
Not sure what you are asking for. It already remembers the position in the media browser. If you go into spotify playlists, then refresh the page you'll still be there.

Search is not available as far as I know from the home assistant backend. Correct me if I am wrong.
